plugin/src/main/java/org/jetbrains/research/refactorinsight/actions/ComboBoxRefactoringAction.java

```diff
@@ -0,0 +1,128 @@
+package org.jetbrains.research.refactorinsight.actions;
+
+import com.intellij.openapi.actionSystem.*;
+import com.intellij.openapi.actionSystem.ex.ComboBoxAction;
+import com.intellij.openapi.project.DumbAware;
+import com.intellij.openapi.project.Project;
+import com.intellij.ui.JBColor;
+import com.intellij.util.ui.JBInsets;
+import com.intellij.util.ui.JBUI;
+import com.intellij.vcs.log.ui.MainVcsLogUi;
+import com.intellij.vcs.log.ui.VcsLogInternalDataKeys;
+import org.jetbrains.annotations.Nls;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.research.refactorinsight.services.WindowService;
+
+import javax.swing.*;
+import java.awt.*;
+import java.util.Arrays;
+
+import static java.awt.Font.*;
+
+public class ComboBoxRefactoringAction extends ComboBoxAction implements DumbAware {
+
+    private enum Tab {
+        FILES("Files"),
+        REFACTORING("Refactorings");
+        public final String label;
+        Tab(String label) {
+            this.label = label;
+        }
+    }
+
+    private DefaultActionGroup myActions;
+
+    private Tab currentTab = Tab.FILES;
+
+    @Override
+    public @NotNull ActionUpdateThread getActionUpdateThread() {
+        return ActionUpdateThread.EDT;
+    }
+
+    @Override
+    public void update(@NotNull AnActionEvent e) {
+        Presentation presentation = e.getPresentation();
+        presentation.setText(getText(getValue()));
+    }
+
+    @Override
+    public @NotNull JComponent createCustomComponent(@NotNull Presentation presentation, @NotNull String place) {
+        JPanel panel = new JPanel(new GridBagLayout());
+        panel.setBorder(BorderFactory.createEmptyBorder(0, 6, 0, 6));
+        ComboBoxButton button = createComboBoxButton(presentation);
+        button.setBorder(BorderFactory.createEmptyBorder());
+        button.setForeground(JBColor.BLUE);
+        button.setMargin(JBUI.emptyInsets());
+        JLabel label = new JLabel("Show:");
+        label.setFont(new Font("Default", PLAIN, 11));
```

Comment:
Maybe we should make it a bit bigger? BTW, is there a way to choose font size used in IDE? It might look weird if a user uses font size equal to 16 for IDE, but we show 11 for our list instead.

Reply:
If not specify any size, it looks like this:
<img width="217" alt="Screenshot 2023-04-21 at 19 07 03" src="https://user-images.githubusercontent.com/57748412/233694974-f070eb90-2a9d-47fc-a96f-31c0c4c1dc97.png">

I thought that the size for the button text and the size for the label should be the same (as on the mockup), and the size of the button text is 11, so I wrote 11. Now I have changed that the size is taken from the button. But the size can be omitted, in which case it will default to 16.

Reply:
Agree, they should be the same size